source/basic.tex

```diff
@@ -2520,7 +2520,7 @@
 s)~(\ref{multibyte.strings}) and \tcode{argv[0]} shall be the pointer to
 the initial character of a \ntmbs that represents the name used to
 invoke the program or \tcode{""}. The value of \tcode{argc} shall be
-non-negative. The value of \tcode{argv[argc]} shall be 0. \begin{note} It
+non-negative. The value of \tcode{argv[argc]} shall be \tcode{nullptr}. \begin{note} It
 is recommended that any further (optional) parameters be added after
```

Comment:
Hm, well, `argv[argc]` is of type `char*`, not `std::nullptr_t`, so I'm not sure this is accurate. It wasn't accurate in the first place, but perhaps we should fix that instead. ("... shall be a null"?)